### ISSUE TITLE
Horizonal shrink warp Textfield

### ISSUE THREAD

**Issue by jascodes** [flutter/flutter#39949]
As you can see below I want implement text field of variable size. Currently I intend to use stack and look for way to get hold of renderbox of text and find out it that is feasible.

![Payroll 2](https://user-images.githubusercontent.com/1231593/64394275-feba9280-d072-11e9-86d6-5247d72c3d4d.png)

Thx

**Comment by BondarenkoStas** [comment]
This is a question better suited for StackOverflow, since there doesn’t seem to be anything wrong with Flutter itself, but related to implementation. Closing, as this is not an issue report. See https://flutter.dev/community for support options.
If you disagree, feel free to comment and I will reopen the issue.

**Comment by jascodes** [comment]
@BondarenkoStas I disagree, I posted here as I think its not possible outofbox, and this is feature request.

Thx

**Comment by goderbauer** [comment]
What exactly is the feature request here? Not sure I understand what "textfield of variable size" exactly means.

**Comment by jascodes** [comment]
@goderbauer So the size of input field expanding as we type in; based on the space required by input text type. In the attached screenshot, you maybe see currency code on the left hand side. Its very tricky to implement without variable width.

Thxc

**Comment by jascodes** [comment]
Maybe custom implementation of text field is required in this case. Any one know any resources/snippets in regart to this?
